explore/scripts/get_repos_users.py

```diff
@@ -31,7 +33,7 @@
 
 # Iterate through internal repos
 print("Gathering data across multiple paginated queries...")
-collective = {u'data': {}}
+collectiveExt = {u'data': {}}
```

Comment:
Do you need a setup of `collectiveInt` here? I don't see the creation of that objection anywhere... Just usage down in line 65, which is going to break I suspect.

Reply:
Line 20.
Already ran a full update #80 , no breakage.